modules/pop3/modules.php

```diff
@@ -0,0 +1,1129 @@
+<?php
```

Comment:
I think we removed the pop3 module. No need for converting it to bootstrap.

Reply:
Indeed: https://github.com/cypht-org/cypht/pull/614

Any POP leftovers should be removed.

Reply:
All right, Thanks